contact.html

```diff
@@ -0,0 +1,622 @@
+                                      <div class="cmp-form-text" data-cmp-required-message="You need to add your email to log in">
+                                        <input class="mail" type="email" id="email" name="email" placeholder="Email">
+                                      </div>
+                                    </div>
+                                    <div class="text phone-field aem-GridColumn aem-GridColumn--default--6">
+                                      <div class="cmp-form-text">
+                                        <input class="phone" type="tel" id="phone" name="phone" pattern="[0-9]{3}-[0-9]{2}-[0-9]{3}" placeholder="Phone">
+                                      </div>
+                                    </div>
+
+                                    <div class="cmp cmp-options aem-GridColumn aem-GridColumn--default--12">
+                                      <div class="form-group drop-down">
+                                        <select id="country" name="country">
+                                          <option value="" disabled selected>Location</option>
+                                          <option value="australia">Australia</option>
+                                          <option value="canada">Canada</option>
+                                          <option value="usa">USA</option>
+                                        </select>
+                                      </div>
+                                    </div>
+
+                                    <div class="cmp cmp-options aem-GridColumn aem-GridColumn--default--12">
+                                      <div class="form-group drop-down">
+                                        <select id="country" name="country" >
+                                          <option value="" disabled selected>Reason for Inquiry</option>
+                                          <option value="australia">Australia</option>
+                                          <option value="canada">Canada</option>
+                                          <option value="usa">USA</option>
+                                        </select>
+                                      </div>
+                                    </div>
+
+                                    <div class="text aem-GridColumn aem-GridColumn--default--12">
+                                      <div class="cmp-form-text">
+                                        <textarea id="subject" name="subject" placeholder="How can we help you?" style="height:200px"></textarea>
+                                      </div>
+                                    </div>
+
+                                    <div class="button aem-GridColumn aem-GridColumn--default--12">
+                                      <button type="BUTTON" id="form-button-1456471026" class="cmp-form-button submit" name="button">Send</button>
```

Comment:
@Prystal254 `.submit` can't go here. Custom classes must go on the root element of the component.